pokemon-checker/src/constants/Time.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Constants for measureing time and calculating timestamps
+ */
+export const enum Time {
+  MILLISECONDS_PER_SECOND = 1000,
+  SECONDS_PER_MINUTE = 60,
+  MINUTES_PER_HOUR = 60,
+}
```

Comment:
This is good, remind me to talk about structure next time we have a chat because I added a utils directory, I'm not sure if we can double down on this or keep things seperate